def my_test_func():
    print("Вызов функции успешный")


